ios/Classes/FacebookModule.m

```diff
@@ -359,6 +391,10 @@ -(void)authorize:(id)args
     BOOL allowUI = args == nil ? YES : NO;
     NSArray *permissions_ = permissions == nil ? [NSArray array] : permissions;
     FBSDKLoginManager *loginManager = [[[FBSDKLoginManager alloc] init] autorelease];
+    [loginManager setLoginBehavior:FBSDKLoginBehaviorBrowser];
```

Comment:
Correct me if I'm wrong but FBSDKLoginBehaviorNative is the default behaviour.

 The original issue was that it did not open up the authorize screen without the behaviour being set explicitly (to browser, native still won't work(?)).